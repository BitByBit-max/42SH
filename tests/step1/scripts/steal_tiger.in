echo trying to steal tiger reference...
if false; then true; else echo hi; fi
