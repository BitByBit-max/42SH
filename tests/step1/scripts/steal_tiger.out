trying to steal tiger reference...
hi
